msteams-platform/tabs/how-to/authentication/auth-aad-sso.md

```diff
@@ -201,11 +201,11 @@ IConfidentialClientApplication app = ConfidentialClientApplicationBuilder.Create
 
 ### [Node.js](#tab/nodejs)
 
-```javascript
+```Node.js
```

Comment:
Node.js is not a language. Shouldn't this be javascript to be properly highlighted?